modules/client-handler/src/test/java/org/apache/ignite/client/handler/JdbcQueryEventHandlerImplTest.java

```diff
@@ -123,23 +122,21 @@ public void sessionRecreatedIfExpired() throws IgniteInternalCheckedException {
 
         JdbcConnectionContext context = resourceRegistry.get(connectionId).get(JdbcConnectionContext.class);
 
-        when(queryProcessor.createSession(any())).thenAnswer(inv -> new SessionId(UUID.randomUUID()));
+        SessionId sessionId = new SessionId(UUID.randomUUID());
 
-        List<Integer> errorCodes = List.of(Sql.SESSION_EXPIRED_ERR, Sql.SESSION_NOT_FOUND_ERR);
+        when(queryProcessor.createSession(any())).thenAnswer(inv -> sessionId);
```

Comment:
```suggestion
        when(queryProcessor.createSession(any())).thenReturn(sessionId);
```

Reply:
Do we really need this mock?

Reply:
It was there before but it is not used. Removed, thanks.